addons/l10n_eu_oss/models/res_company.py

```diff
@@ -45,7 +45,10 @@ def _map_eu_taxes(self):
             oss_countries = eu_countries - company.account_fiscal_country_id - multi_tax_reports_countries_fpos.country_id
             tg = self.env['account.tax.group'].search([
                 *self.env['account.tax.group']._check_company_domain(company),
-                ('tax_payable_account_id', '!=', False)], limit=1)
+                ('tax_payable_account_id', '!=', False),
+                ('tax_payable_account_id.code', '!=', False)], limit=1)
+            if not tg:
+                continue
```

Comment:
```suggestion
```

Here, create or update fiscal positions for each EU country, so we should fix this issue while looping the company.
```py
if company.id not in eu_countries.ids:
    continue
```